packages/workbox-precaching/test/sw/cookies.js

```diff
@@ -1,20 +1,38 @@
-importScripts('/__test/mocha/sw-utils.js');
-importScripts('/__test/bundle/workbox-precaching');
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
 
-describe('Test Cookies with Precache', function() {
-  it('should cache asset with appropriate cookies with revisions asset', function() {
-    const revManager = new workbox.precaching.RevisionedCacheManager();
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/* eslint-env mocha, browser */
+/* global expect */
+
+import RevisionedCacheManager
```

Comment:
What is the advantage of this vs testing the actual build files here?

I follow that for developing the tests with stubs etc, bundling is useful, but here we've switch from testing the dev and prod builds to a custom test build, which isn't necessarily bad, it's not going to cover some edge cases we've hit with the actual end builds.